### PR TITLE
Molecules > LinkCard and RootPortal

### PR DESCRIPTION
**Pull request by wintondeshong**
- [x] Related GitHub issue(s) linked in PR description
- [x] Destination branch merged, built and tested with your changes
- [x] Code formatted and follows best practices and patterns
- [x] Code builds cleanly (no _additional_ warnings or errors)
- [x] Manually tested
- [x] Automated tests are passing
- [x] **Intentional during scaffolding** No _decreases_ in automated test coverage
- [x] Documentation updated (readme, docs, comments, etc.)
- [x] Localization: No hard-coded error messages in code files (_minimally_ in string constants)
- [x] New component and/ or properties? Make sure to update storybook

### CODE REVIEW

File: src/molecules/link-card/link-card.tsx [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+// #region Interfaces
+// -------------------------------------------------------------------------------------------------
+
+export interface LinkCardProps {
+    children: any;
+    includeIcon?: boolean;
+    label: string;
+    to?: any;
+    type?: LinkCardTypes;
+    onClick?: () => void;
```
Comment (jhugs) [inline src/molecules/link-card/link-card.tsx:29]: minor: Alphabetization (if we do in these projects)

File: src/molecules/link-card/link-card.tsx [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+        cssClassNames.push("-with-icon");
+    }
+    const cssClassNamesFlat = cssClassNames.join(" ");
+
+    const type = props.type ?? LinkCardTypes.Link;
+
+    const renderChildren = () => (
+        <React.Fragment>
+            {// if
+            props.includeIcon && <Icon type={Icons.Lightbulb} />}
```
Comment (jhugs) [inline src/molecules/link-card/link-card.tsx:50]: should this icon be configurable via a prop instead of a boolean?

Comment (mrjones2014) [inline src/molecules/link-card/link-card.tsx:50]: the icon type should probably be a prop

Comment (wintondeshong) [reply src/molecules/link-card/link-card.tsx:50]: I'll make the type optionally configured and defaulted to `LightBulb`

**Comment by brandongregoryscott** [comment]
@all-contributors add @mrjones2014 for review

**Pull request merged**